node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
I like idea of API parity with Johnny-Five since it has real world testing behind it. I can change `isOn` into a defined property and add the proper tests around that implementation. Should attempting to set that property throw an Error?